jina/hubble/helper.py

```diff
@@ -496,13 +542,14 @@ def _get_install_options(requirements_file: 'Path', excludes: Tuple[str] = ('jin
             if (not req) or req.startswith('#'):
                 continue
             elif req.startswith('-'):
-                install_options.extend(req.split(' '))
+                for index, item in enumerate(req.split(' ')):
+                    install_options.append(expand_env_variables(item))
             else:
+                expand_req = expand_env_variables(req)
                 req_spec = parse_requirement(req)
```

Comment:
Should we use `expand_req` here?

Reply:
yes we can remove it now

Reply:
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/16202568/182845464-8c79a20d-7dfe-43c2-beac-9caee403102e.png">
in local test is success, but failed  in github action

Reply:
I mean `req_spec = parse_requirement(req)` -> `req_spec = parse_requirement(expand_req)`

Reply:
yes,  we should use expand_req,  prevent parse_requirement function from parsing `{}` error